wri/wri-odp
# Approval UI

This PR implemented approval UI based on odp-190

to test

* Go to ckan backend create dataset with `draft` click and `approval status=appending` 
* to see diff when you go to approve request page, use the api `api/action/pending_dataset_create` to create a pending dataset.
    * `package_id`: id of the dataset 
    *  `package_data`: dataset metadata: you can copy them from`package_show` ; 
    * edit one or two metadata: `resource description`, etc.
* go to the frontend dashboard `dashboard/approval-request` and then interact with the page
* click review to go to the dataset page, toggle between buttons
* go to awaiting approval

<img width="1726" alt="approv1" src="https://github.com/wri/wri-odp/assets/20904032/97e627dd-2fab-45d8-a4f4-9c3745344439">


<img width="1378" alt="qpprov2" src="https://github.com/wri/wri-odp/assets/20904032/d23aad2f-9055-4005-838e-bf4111971c57">

Toggle between
<img width="1464" alt="approv5" src="https://github.com/wri/wri-odp/assets/20904032/dca42689-ba8c-403a-9fd7-85476739ff3e">
<img width="1421" alt="approv6" src="https://github.com/wri/wri-odp/assets/20904032/7d610482-a68c-4120-8e02-620bb49ea6a7">

**NOTE**: no layer handled here, will be done in the next ticket
